src/Application.php

```diff
@@ -66,8 +67,11 @@ protected function getDefaultInputDefinition()
             new InputOption('--quiet', '-q', InputOption::VALUE_NONE, 'Do not output any message'),
             new InputOption('--verbose', '-v|vv|vvv', InputOption::VALUE_NONE, 'Increase the verbosity of messages'),
             new InputOption('--version', '-V', InputOption::VALUE_NONE, 'Display this application version'),
-            new InputOption('--yes', '-y', InputOption::VALUE_NONE, 'Answer "yes" to confirmation questions; accept the default value for other questions; disable interaction'),
-            new InputOption('--no', '-n', InputOption::VALUE_NONE, 'Answer "no" to confirmation questions; accept the default value for other questions; disable interaction'),
+            new InputOption('--no-interaction', null, InputOption::VALUE_NONE, 'Do not ask any interactive questions'),
```

Comment:
Would it make sense to make it explicit that it accepts the defaults?
```suggestion
            new InputOption('--no-interaction', null, InputOption::VALUE_NONE, 'Do not ask any interactive questions; accept default values'),
```

Reply:
Also, I guess there's not a shorter version of the flag? Seems unfortunate to be able to shorten `yes` but not `no-interaction` :grinning:

Reply:
That's the tricky part, `-n` is already a shortcut for `--no-interaction` in Symfony Console by default, but in this CLI it's the shortcut for `--no`.

In this PR the behavior would be:

- `--no-interaction`: accept default values (including for confirmation questions, whose default is usually yes but sometimes no)
- `--yes` / `-y`: answer yes to confirmation questions, otherwise accept default values
- `--no` / `-n`: answer no to confirmation questions, otherwise accept default values

Reply:
With https://github.com/platformsh/platformsh-cli/pull/1183, I believe `-y` would have the same effect as `--no-interaction`.